how-to/customize-workspace/client/src/modules/auth/example/shapes.ts

```diff
@@ -0,0 +1,31 @@
+export interface ExampleUser {
+	name: string;
+	email: string;
+	role: string;
+}
+
+export interface ExampleOptions {
+	autoLogin: boolean;
+	authenticatedUrl: string;
+	loginUrl: string;
+	logoutUrl: string;
+	loginHeight: number;
```

Comment:
Some of these properties should be optional

Reply:
done